Source/Generators/LMCArraySignalGenerator.cc

```diff
@@ -563,6 +563,12 @@ namespace locust
             return false;
         }
 
+        if(!fHilbertTransform.SetupHilbertTransform())
+        {
+            LERROR(lmclog,"Error initializing Hilbert transform.\n");
+            return false;
+        }
+

```

Comment:
As mentioned above, can this functionality happen from within HilbertTransform::Configure()?  Or does it need to happen from outside HilbertTransform?